azkaban-web-server/src/main/java/azkaban/imagemgmt/daos/ImageTypeDao.java

```diff
@@ -0,0 +1,9 @@
+package azkaban.imagemgmt.daos;
+
+import azkaban.imagemgmt.models.ImageType;
+import java.util.Optional;
+
+public interface ImageTypeDao {
```

Comment:
Can you please move Image DAO and models in azkaban-common module? Dispatch logic is in common and I will need these DAOs and APIs over there.

Reply:
Thanks for pointing this out. Moved DAOs, models, exceptions in the azkaban-common so that dispatch logic can leverage the same.